### DISCUSSION TITLE
Is Any one face this problem ?

### DISCUSSION THREAD

**Discussion by yhak22** [skyline-emu/skyline#574] [Q&A] [answered]
Hi guys .

is any one have problem in celeste or sourge bringer games
i mean when i first play it the frame is stable at 30fps but after 5 minutes the fps slow down until reach 2fps this happens every time i play those games  but in sonic mania it's stable at 60fps
so i wonder if this is known issue or something
 my device is SD865 and 8GB Ram
thank you guys. :) 👍

**Comment by PixelyIon** [comment, marked as answer]
This is expected and an issue we're actively working on fixing.

**Comment by yhak22** [comment]
thanks man i love your hard work :)
